Source/Engine/Physics/Colliders/CharacterController.h

```diff
@@ -141,6 +142,16 @@ DECLARE_SCENE_OBJECT(CharacterController);
     /// </summary>
     API_PROPERTY() void SetStepOffset(float value);
 
+    /// <summary>
+    /// Sets the character up vector.
+    /// </summary>
+    API_PROPERTY() void SetUpDirection(const Vector3& up);
+
+    /// <summary>
+    /// Gets the character up vector.
+    /// </summary>
+    API_PROPERTY() Vector3 GetUpDirection() const;
```

Comment:
Move getter up to be first (in order). And add attributes like: `Attributes="EditorOrder(240), DefaultValue(true), EditorDisplay(\"Character Controller\")"`